lib/HLSL/DxilContainerReflection.cpp

```diff
@@ -1085,7 +1085,7 @@ HRESULT CShaderReflectionType::Initialize(
 
       // Try to "clean" the type name for use in reflection data
       llvm::StringRef name = structType->getName();
-      name = name.ltrim("dx.alignment.legacy.");
+      name = name.ltrim(kHostLayoutTypePrefix);
```

Comment:
Should this keep both old and new trim in case IR is from prior version of compiler?

Reply:
done